my_enumerable.rb

```diff
@@ -0,0 +1,24 @@
+# Create MyEnumerable module
+module MyEnumerable
+  def my_all?
+    each do |element|
+      return false unless yield(element)
+    end
+    true
+  end
+
```

Comment:
- [ ] Here is the result I get when I verify your solution: 👇🏻 Kindly fix this. I'l recommand you add instruction on how to run your code into the README file.

![image](https://github.com/shayan1234554321/enumerable/assets/79058364/490168ae-0910-4abe-a847-b7cb1ab276a7)